routes/seaice.py

```diff
@@ -0,0 +1,214 @@
+import asyncio
+import numpy as np
+from math import floor
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+)
+
+# local imports
+from fetch_data import (
+    fetch_wcs_point_data,
+    get_dim_encodings,
+    deepflatten,
+    build_csv_dicts,
+    write_csv,
+)
+from validate_request import (
+    validate_seaice_latlon,
+    project_latlon,
+    validate_seaice_year,
+)
+from validate_data import nullify_and_prune, postprocess
+from . import routes
+from config import WEST_BBOX, EAST_BBOX
+
+seaice_api = Blueprint("seaice_api", __name__)
+# Rasdaman targets
+seaice_coverage_id = "hsia_arctic_production"
+
+
+def create_csv(data_pkg, lat=None, lon=None):
+    """Create CSV file with metadata string and location based filename.
+    Args:
+        data_pkg (dict): JSON-like object of data
+        lat: latitude for points or None for polygons
+        lon: longitude for points or None for polygons
+    Returns:
+        CSV response object
+    """
+
+    fieldnames = [
+        "year",
+        "month",
+        "concentration"
+    ]
+
+    months = [
```

Comment:
No need to change it, but I just realized from working on something else that you can do:
```python
import calendar
months = list(calendar.month_name)
```
to generate a list of month names.

Reply:
Glad to know this exists -- I always consider it kind of a "bad smell" when I enumerate months in code!  I'd suggest changing it just to remove the hard coded months.